client/jetpack-cloud/sections/partner-portal/payment-method-actions/index.tsx

```diff
@@ -35,7 +42,7 @@ const PaymentMethodActions: FunctionComponent< Props > = ( { card } ) => {
 			.catch( ( error: Error ) => {
 				reduxDispatch( errorNotice( error.message ) );
 			} );
-	}, [ closeDialog, card, translate, reduxDispatch ] );
+	}, [ closeDialog, card, translate, reduxDispatch, nextPrimaryPaymentMethod ] );
```

Comment:
The callback only depends on nextPrimaryPaymentMethod?.id so let's use that instead in the dependency array.

Reply:
Thanks! Updated in e121f57fab0a56c5653bf4d67a99a44958db2a81